include/sddf/i2c/queue.h

```diff
@@ -41,17 +41,23 @@ enum i2c_token {
     I2C_TOKEN_END = 0x0,
     /* START: Begin a transfer. Causes master device to capture bus. */
     I2C_TOKEN_START = 0x1,
-    /* ADDRESS WRITE: Used to wake up the target device on the bus.
+    /* ADDRESS WRITE: Address target and unset READ bit.
      * The byte immediately following this token is an integer (N) length of the succeeding
-       write. Max 256 (1**8). The next N bytes are the payload. */
+       write. Max 256 (1<<8). The next N bytes are the payload. */
```

Comment:
```suggestion
       write. Max 255 (1<<8). The next N bytes are the payload. */
```

Reply:
It's 255 right?

Reply:
Yes it is actually, but maybe it shouldn't be. There's no reason to represent a read or write of length 0